runtimevar/etcdvar/etcdvar_test.go

```diff
@@ -38,20 +39,25 @@ var (
 )
 
 func init() {
-	etcdClient, etcdError = clientv3.NewFromURL("http://localhost:2379")
-	if etcdError == nil {
-		// Check to see if the local etcd is actually running.
-		ctx, cancel := context.WithTimeout(context.Background(), 3*time.Second)
-		defer cancel()
-		_, etcdError = etcdClient.Put(ctx, "unused", "unused")
+	if setup.HasDockerTestEnvironment() {
+		etcdClient, etcdError = clientv3.NewFromURL("http://localhost:2379")
+		if etcdError == nil {
```

Comment:
Lines 44-48 can be dropped; same as above, tests will just fail naturally if etcd isn't running.